CruzHacks/cruzhacks-2021-website
# Add new sponsors.

Problem
=======
I want to update the sponsors section to include our new sponsors.

Solution
========
I added the Santa Cruz Works, Progress, Blackstone and CROSS logos and rearranged the logos grid according to the newer Figma design.

Steps to Verify:
----------------
1. Checkout the `chore/update-sponsor-logos-v2` branch
2. Verify that the sponsor logos look good on all viewports
3. Also verify the links

Screenshots (optional):
-----------------------

![image](https://user-images.githubusercontent.com/20177171/104541804-2c084080-55d7-11eb-9b14-b27ca71cef62.png)

![image](https://user-images.githubusercontent.com/20177171/104541869-4b9f6900-55d7-11eb-8f09-215fd18c08c2.png)
